### PR TITLE
docker updates

### PR DESCRIPTION
**Pull request by koning**
Fix docker user.
Change MANIFEST.in to use recursive-include.
Update docker docs.
Add port to rabbitmq config.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +23,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   value can be None, default rabbitmq broker, or redis for a redis
   local broker.
 - Missing doc options for run and run-workers.
+- Add port to rabbitmq config options.
+- Updated docker docs.
```
Comment (ben-bay) [inline CHANGELOG.md]: This is more of a `Changed` item in my opinion

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated the Merlin Sphinx web docs.
 - Changed the example workflows to use python3 instead of python.
 - Updated `merlin info` to lookup python3 and and pip3.
+- Altered user in Dockerfile and removed build tools.
+- MANIFEST.in now uses recursive-include.
+- Added author and author_email to setup.py.
```
Comment (ben-bay) [inline CHANGELOG.md]: Move to `Added`

File: setup.py [hunk 3]
```diff
@@ -91,6 +91,8 @@ def extras_require():
 
 setup(
     name="merlinwf",
+    author="Merlin Dev team",
+    author_email="merlin-dev@llnl.gov",
```
Comment (ben-bay) [inline setup.py]: @lucpeterson is this email just supposed to be `merlin@llnl.gov`?

File: CHANGELOG.md (hunk 1 above)
Comment (koning) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (koning) [reply CHANGELOG.md]: Fixed

**Pull request merged**